api_v3/lib/types/resource/KalturaUploadedFileTokenResource.php

```diff
@@ -7,7 +7,7 @@
  * @see api/services/UploadService#uploadAction()
  * @see api/services/UploadTokenService#addAction()
  */
-class KalturaUploadedFileTokenResource extends KalturaDataCenterContentResource
+class KalturaUploadedFileTokenResource extends KalturaDataEntryAllowedContentResource
```

Comment:
Rename KalturaDataEntryAllowedContentResource to KalturaGenericDataCenterContentResource